### ISSUE TITLE
Chest and item names are not localized when unloading and managing/displaying blacklist

### ISSUE THREAD

**Issue by rutzfutz** [JEFF-Media-GbR/Spigot-InvUnloadPlus#41]
I noticed that you have a command `/searchitem <item>` that displays properly localized item names. Unfortunately, these localized names are not used by the main functions like `/unload`, `/dump`, `/unloadinfo` etc., or the blacklist functions.

It would be great if the plugin could use the localized names wherever possible to provide a consistent experience to non-English players.

My InvUnload version is 4.9.4 running on PaperMC build 484 (MC: 1.16.5).

**Comment by mfnalex** [comment]
I have localized display names?! I don't remember adding that. I'll have a look how I did that after I am done with my current work :)

**Comment by rutzfutz** [comment]
I'm sorry, I just realized that the command `/searchitem` that I executed was actually not your `/invunload:searchitem` command, but `/worldedit:searchitem`.

WorldEdit seems to be using some kind of registry to implement this command:
https://github.com/EngineHub/WorldEdit/blob/master/worldedit-core/src/main/java/com/sk89q/worldedit/command/GeneralCommands.java

I also found this in the API docs, but haven't tested it myself (I'm not a Java/plugin developer... yet):
https://hub.spigotmc.org/javadocs/spigot/org/bukkit/inventory/meta/ItemMeta.html#getLocalizedName()

Bottom line: It would be great to have localized item names in your plugin, too. But I completely understand if this is too complicated. As I created this issue, I incorrectly assumed you'd already have the localization implemented, so I thought it could be a quick fix as well. Feel free to close this if it would take too much time.

**Comment by mfnalex** [comment]
The problem is that I don't think it is possible to get the translated names server side. It IS possible to send the "regular" to the client while telling the client to "show" the translated versions, but when it's sent back to the server, it cannot understand it. The only possibility would be that I parse all the client translations with every update, but I don't wanna do that for obvious reasons, sorry.

**Comment by mfnalex** [comment]
If you however find any way to do so, feel free to let me know, but I am really like 95% sure it is not possible without some kind of database etc.
